ruby/spec/inclusive_code/cop/inclusive_code_spec.rb

```diff
@@ -92,6 +92,71 @@ class MyInclusiveEngine
       end
     end
 
+    context 'when non-inclusive terms are present in an allowed file' do
+      let(:flagged_terms) do
+        {
+          'flagged_terms' => {
+            'rubocop' => {
+              'suggestions' => ['ruby_enforcement_service'],
+              'allowed' => [],
+              'allowed_files' => ['README.md', 'lib/rubocop/cop/inclusive_code.rb']
+            },
+            'other_offensive_term' => {
+              'suggestions' => ['non_offensive_terms'],
+              'allowed' => [],
+              'allowed_files' => []
+            }
+          }
+        }
+      end
+
+      subject(:cop) do
+        described_class.new(nil, nil, flagged_terms)
+      end
+
+      it 'can ignore offenses within the path of an allowed file while detecting the'\
+        'same offenses in the path of another file' do
```

Comment:
```suggestion
```

Reply:
Just wanted to share that you can leave a multi-line suggestion by (while holding shift) clicking the top line then the bottom line to select a range of lines. From there you click to add a comment and insert the suggestion as usual.